metadata.rb

```diff
@@ -22,7 +22,7 @@
 license          'Apache-2.0'
 description      'Installs and configures operating system hardening'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '3.1.0'
```

Comment:
@ekelson-bcove please do not bump the cookbook version. This is done by maintainers before the release is done